src/clients/go/tb_client_test.go

```diff
@@ -118,9 +118,7 @@ func doTestClient(t *testing.T, client Client) {
 			accountA,
 			accountB,
 		})
-		if err != nil {
-			t.Fatal(err)
-		}
+		assert.True(t, err == nil)
```

Comment:
A minor note is that `assert.True(t, err == nil)` erases the information about _which_ error occurred. It'll tell you that `true != false`, not that you got some specific error. I don't think this is a big deal, but, if we go to the pain of using a third-party assertion library, we might as well actually use it? I think the right incarnation here would be

```
assert.Nil(err)
```

Reply:
But I think I'd stick with `Fatal` here and elsewhere, except the specific cases. 

`Fatal` stops the test here and now, `assert.True`, `assert.Nil` will continue the test. But it doesn't make sense to continue the test if we weren't able to create accounts?

So I suggest maybe just keeping `t.Fatal`, and replacing it with `t.Error` when we spawn coroutines?

Reply:
Yeah, no strong opinion on whether we should abort the test or continue. I do have a strong opinion that `assert.Nil(err)` is better than `assert.True(err == nil)`, as that would be a strict improvement (Nil also continues).